lisa/stats.py

```diff
@@ -840,6 +848,35 @@ def split_row_col(group):
             collapsed_col = None
             collapse_group = {}
 
+        if backend == 'plotly':
+            figure = plotly_make_subplots(
+                rows=nrows, cols=ncols,
+                subplot_titles=[
+                    '-'.join(group) if isinstance(group, tuple) else group
+                    for group, subdf in grouped
+                ]
+            )
+
+            for group, subdf in grouped:
+                group = fixup_tuple(group)
+                row, col = split_row_col(group)
+                group_dict = dict(zip(group_on, group))
+                x = rows.index(row) + 1
+                y = cols.index(col) + 1
+                subdf = subdf.drop(columns=group_on)
+                subdf = self._collapse_cols(subdf, collapse_group)
+                group_dict = dict(zip(group_on, group))
+                plot_func(subdf, (figure, x, y), collapsed_col,
```

Comment:
should use named parameters, the risk of confusion becomes to great with signatures that long.